src/sql/workbench/common/workspaceActions.ts

```diff
@@ -5,7 +5,7 @@
 
 import { Action } from 'vs/base/common/actions';
 import { IOpenerService } from 'vs/platform/opener/common/opener';
-// eslint-disable-next-line code-layering
+// eslint-disable-next-line code-layering,code-import-patterns
```

Comment:
Are these layering issues something we need to fix up at some point for the work you're doing? If so are we tracking these anywhere?